README.md

```diff
@@ -1,6 +1,17 @@
-# CSC510-22-sans-name
-Repo for Software Engineering Group Project - Fall 2022
+# CSC510-22-group-5
 
-![unit test workflow](https://github.com/alunavat/CSC510-22-sans-name/actions/workflows/unit-test.yml/badge.svg)
+Repo for Software Engineering Group HW - Fall 2022 - Group 5
+
+This repo will be used throughout the CSC510 course to showcase the team contribution towards weekly assignments.
+
+![Unit test workflow](https://github.com/alunavat/CSC510-22-sans-name/actions/workflows/unit-test.yml/badge.svg)
```

Comment:
The URL here references the old project name. Needs an update.

Reply:
Done!